includes/class-wp-job-manager-stats.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * File containing the class WP_Job_Manager_Stats
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * This class is responsible for initializing all aspects of stats for wpjm.
+ */
+class WP_Job_Manager_Stats {
+	/**
+	 * Holds the one instance of stats.
+	 *
+	 * @var null|WP_Job_Manager_Stats
+	 */
+	private static $instance = null;
+
+	/**
+	 * Constructor.
+	 */
+	private function __construct() {
+		$this->init();
+	}
+
+	/**
+	 * Get the instance.
+	 *
+	 * @return WP_Job_Manager_Stats
+	 */
+	public static function instance(): WP_Job_Manager_Stats {
+		if ( null === self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
```

Comment:
WDYT about using the `Singleton` trait that we already have for this? 

https://github.com/Automattic/WP-Job-Manager/blob/trunk/includes/trait-singleton.php